content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md

```diff
@@ -8,10 +8,12 @@ card:
 ---
 
 <!-- overview -->
+Many applications rely on configuration which is used during either application initialization or runtime.
+Most of the times there is a requirement to adjust values assigned to configuration parameters.
+ConfigMaps is the kubernetes way to inject application pods with configuration data. 
```

Comment:
Watch out for the grammar and capitalization here (“ConfigMaps is”; “kubernetes”).

Reply:
@sftim  @divya-mohan0209  Thanks for your inputs. How about below text?

Many applications rely on configuration which is used during either application startup or runtime. Most of the time, users also want to adjust the configuration settings after the initial setup.ConfigMap is a great way to inject application pods with configuration data which can be consumed by containers during initialization or runtime.